Models.py

```diff
@@ -0,0 +1,66 @@
+"""
+Define all data models.
+"""
+
+from py2neo.ogm import (GraphObject, Property, Related, RelatedTo, RelatedFrom)
+
+
+class Person(GraphObject):
+    __primarykey__ = "uid"
```

Comment:
I'm not sure we need this because uniqueness will be determined through username.